readthedocs/oauth/migrations/0011_add_remote_relation_model.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 2.2.16 on 2020-10-05 06:10
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('oauth', '0010_index_full_name'),
+    ]
+
+    operations = [
+        migrations.SeparateDatabaseAndState(
+            state_operations=[
+                migrations.CreateModel(
+                    name='RemoteRelation',
+                    fields=[
+                        ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                    ],
+                    options={
+                        'db_table': 'oauth_remoterepository_users',
```

Comment:
The example does not use `db_table` here but uses `database_operations` and renames the table. Does this have the same effect?

Reply:
Yes, It should have the same effect, but After seeing the example from Django it seems better to rename the model. I will test that out.